shells/backends/gcp_tf_backend/src/driver.py

```diff
@@ -0,0 +1,295 @@
+                    os.environ[GOOGLE_APPLICATION_CREDENTIALS] = DYNAMIC_JSON
+                # Keys not defines on AWS TF BACKEND RESOURCE (CLP reference should have been set)
+                else:
+                    # CLP had not been set...
+                    if not gcp_backend_resource.cloud_provider:
+                        self._handle_exception_logging(logger, "At least one method of authentication should be filled")
+                    # Check a correct CLP has been reference
+                    # clp_details = api.GetResourceDetails(aws_backend_resource.cloud_provider)
+                    # print(f"gcp_backend_resource: {gcp_backend_resource.cloud_provider}")
+                    # self._handle_exception_logging(logger, f"gcp_backend_resource: {gcp_backend_resource.cloud_provider}")
+                    # logger.exception(f"gcp_backend_resource: {gcp_backend_resource.cloud_provider}")
+                    # if gcp_backend_resource.cloud_provider == GCP1G_MODEL:
+                    # GCP1G_MODEL:
+                    clp_details = self._validate_clp(api, gcp_backend_resource, logger)
+                    if clp_details.ResourceModelName == GCP1G_MODEL:
+                        print(clp_details.ResourceModelName)
+                        myactual=api.GetResourceDetails(clp_details.Name)
+                        clp_json_path = self._fill_backend_sercret_vars_data(myactual, GCP1G_MODEL)
+                        # account_keys = self._get_storage_keys(api, azure_backend_resource, clp_details)
+                        # model_prefix = ""
+                        # json_path = self._get_attrbiute_value_from_clp(
+                        #     clp_details.ResourceAttributes, model_prefix, CREDENTIALS_JSON_PATH)
+                        # self._handle_exception_logging(logger, f"json_pa {json_path}")
+                        # logger.exception(f"json_path: {json_path}") 
+                        # with open('C:\\out.txt', 'w') as f:
+                        #     with redirect_stdout(f):
+                        #         print('data')
+                        #         print(f"JSON path: {json_path}")
+                        #         print(f"gcp_backend_resource: {gcp_backend_resource.cloud_provider}")  
+                        # if json_path:
+                        # logger.exception(f"json_path: {clp_json_path}")
+                        # json = "C:\\Users\\CSadmin123456\\CS\\tf-gcp\\prod.json"
+                        # logger.exception(f"def: {json}")
+                        os.environ[GOOGLE_APPLICATION_CREDENTIALS] = clp_json_path
+                        # self._handle_exception_logging(logger, f"CLP: {clp_json_path}  DEF: {json}") 
+                        # os.environ[GOOGLE_APPLICATION_CREDENTIALS] = "C:\\Users\\CSadmin123456\\CS\\tf-gcp\\prod.json"
+            except Exception as e:
+                self._handle_exception_logging(logger, f"There was an issue accessing GCP. {e}")
```

Comment:
using this method here will cause a bug that will shadow the original exception
because "_handle_exception_logging" will raise a new ValueError instead of re-raising the original error